.clang-format

```diff
@@ -0,0 +1,65 @@
+# From https://github.com/PickNikRobotics/roscpp_code_format
+# Using the variant from https://github.com/PickNikRobotics/roscpp_code_format/pull/16 for up-to-date syntax for clang-format 10/noetic
```

Comment:
Is this still correct, or did Erich make more adjustments to it?

Reply:
Should I just remove the tag or add something else instead?